docs/understanding-airbyte/connections/README.md

```diff
@@ -28,3 +49,25 @@ When a scheduled connection is first created, a sync is executed as soon as poss
 * **October 3rd, 2:01pm:** since the last sync was less than 24 hours ago, no sync is run
 * **October 3rd, 5:01pm:** It has been more than 24 hours since the last sync, so a sync is run
 
+## Optional operations
+
+### Airbyte basic normalization
+
+As described by the [Airbyte Protocol from the Airbyte Specifications](../airbyte-specification.md), a replication is composed of source connectors that are transmitting data in a JSON format. It is then written as such by the destination connectors. 
+
+On top of this replication, Airbyte provides the option to enable or disable an additional transformation step at the end of the sync called [basic normalization](../basic-normalization.md). This operation is:
+
+- only available for destinations that supports dbt execution.
+- responsible for automatically generating a pipeline or a DAG of dbt transformation models to convert JSON blob objects into normalized tables.
+- running and applying these dbt models to the data written in the destination.
+
+### Custom sync operations
+
+Further operations can be included in a sync on top of Airbyte basic normalization (or even to replace it completely). Potential applications are:
```

Comment:
I think it's better to point to the operations.md file, and move the list describing the operations there. What do you think?
```suggestion
Further operations can be included in a sync on top of Airbyte basic normalization (or even to replace it completely). See operations.md.
```